02-spatial-data.Rmd

```diff
@@ -212,29 +224,33 @@ For more advanced map making we recommend using a dedicated visualization packag
 - map export 
 -->
 
-<!-- Maybe show also somewhere that `world[0]` produces only a plot of the geometry which is rather useful if you do not want to plot a specific attribute. This way, you can for example dismiss the col = "white"-argument in your Nigeria example. -->
+<!-- Maybe show also somewhere that `world[0]` produces only a plot of the geometry which is rather useful if you do not want to plot a specific attribute. This way, you can for example dismiss the col = "white"-argument in your Nigeria example.
+Sorry for commenting on this again but just to clarify africa[0] selects zero columns but since the geometry column is sticky it won't be dismissed. Neverthess, to be more explicit one should probably use plot(st_geometry(africa))-->
 
 ```{r, eval=FALSE, echo=FALSE}
 plot(africa[0], lwd = 3, main = "Nigeria in context", border = "lightgrey")
 ```
 
 #### Further work
 
-**sf** makes R data objects more closely aligned to the data model used in GDAL and GEOS, in theory making spatial data operations faster.
-The work here provides a taste of the way **sf** operates but there is much more to learn (see Chapter \@ref(spatial-data-operations)). And there is also a wealth of information in the highly recommended vignettes of the package.
+**sf** makes R data objects more closely aligned to the data model used in GDAL and GEOS, in theory making spatial data operations faster <!--I am not sure what you mean-->.
```

Comment:
Fair point - the meaning is hidden. I suggest:

```
**sf** simplifies spatial data objects compared with **sp** and provides a near-direct interface to GDAL and GEOS C++ functions.
In theory this should make **sf** faster than **sp**/**rgdal**/**rgeos**, something that is tested in Chapter 5 for data I/O.
This section introduces **sf** classes in preparation for subsequent chapters which deal with vector data (in particular Chapter \@ref(spatial-data-operations)).
```

I've removed the sentence about vignettes. I believe we mention them elsewhere (there are now 4).